recipes/docopt.cpp/all/conanfile.py

```diff
@@ -78,3 +78,5 @@ def package_info(self):
             self.cpp_info.components["docopt"].system_libs = ["m"]
         if self.settings.compiler == "Visual Studio" and self.options.shared:
             self.cpp_info.components["docopt"].defines = ["DOCOPT_DLL"]
+        if self.options.boost_regex:
+            self.cpp_info.components["docopt"].requires.append("boost::boost")
```

Comment:
When boost receives component support, this might be changed to `boost::regex`. I think.